config/default.yml

```diff
@@ -44,6 +44,12 @@ Solidus/SpreeCalculatorPriceSackDeprecated:
   VersionAdded: '0.1'
   Reference: 'https://github.com/solidusio/rubocop-solidus/issues/29'
 
+Solidus/SpreeDefaultAddressDeprecated:
+  Description: 'Checks if user.default_address or user.default_user_address is used and suggest using user.ship_address or user.default_user_ship_address'
+  Enabled: true
+  VersionAdded: '0.1'
```

Comment:
Instead of specifying the version here, it's better to have the `'<<next>>'` string and then run the `cut_release` rake task as [mentioned in the readme](https://github.com/solidusio/rubocop-solidus#release-a-new-version). In this way, we can also deploy a new version on GH and Rubygems.